04/app.js

```diff
@@ -1,9 +1,28 @@
 const user = {
-    firstName: 'Adam',
-    lastName: 'Nowak',
-    born: {
-        day: '14',
-        month: '04',
-        year: '1985'
-    }
-}
\ No newline at end of file
+	firstName: 'Adam',
+	lastName: 'Nowak',
+	born: {
+		day: '15',
+		month: '03',
+		year: '1985',
+	},
+};
+
+const createAgeString = user => {
+	const day = user.born.day;
+	const month = user.born.month;
+
+	return `${day}/${month}`;
+};
+user.calcAge = function (currentDate, birthYear, user) {
+	const userBirthday = birthYear(user);
+
+	if (currentDate.includes(userBirthday)) {
+		console.log('This person has today birthday');
+	}
+
+	return;
```

Comment:
Warto dopytać w zespole czy pisze się jawnie return bo w zasadzie nie ma takiej potrzeby - jeśli tego nie będzie to JS sam to "doda". W obu przypadkach funkcja zwróci "undefined".